test/unit/alignment/pairwise/alignment_result_test.cpp

```diff
@@ -53,22 +54,24 @@ class align_result_test : public ::testing::Test
 {
 public:
 
-    using res_t   = align_result<T>;
-    using id_t    = meta::at_c<T, 0>;
-    using score_t = meta::at_c<T, 1>;
-    using end_t   = meta::at_c<T, 2>;
-    using begin_t = meta::at_c<T, 3>;
-    using trace_t = meta::at_c<T, 4>;
-    using seq_t   = std::tuple_element_t<0, trace_t>;
+    using id_t     = meta::at_c<T, 0>;
```

Comment:
We want to drop the dependency to meta. I suggest you do:
```suggestion
    using id_t     = decltype(std::declval<T>().id);
```

Reply:
Then `T` has to be of type `detail::align_result_value_type` and not `type_list` anymore, right? I'll adapt the implementation of this.